src/w3d/renderer/ddsfile.cpp

```diff
@@ -19,6 +19,10 @@
 #include <algorithm>
 #include <cstring>
 
+// losely based on http://www.nvidia.com/view.asp?IO=dxtc_decompression_code
+// http://msdn.microsoft.com/library/default.asp?url=/library/en-us/dx8_c/hh/dx8_c/graphics_using_0j03.asp
+// http://msdn.microsoft.com/library/default.asp?url=/library/en-us/dx8_c/directx_cpp/Graphics/ProgrammersGuide/Appendix/DDSFileFormat/ovwDDSFileFormat.asp
```

Comment:
I expect these links will be dead in a few months or years. If docs are meant to be saved, it is better to download the page and put it somewhere.